src/core/defaults.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Sets the core defaults
+ */
+export const name = "core/defaults";
+import { rule as checkInternalSlots } from "./linter-rules/check-internal-slots";
+import { rule as checkPunctuation } from "./linter-rules/check-punctuation";
+import linter from "./linter";
+import { rule as localRefsExist } from "./linter-rules/local-refs-exist";
+import { rule as noHeadinglessSectionsRule } from "./linter-rules/no-headingless-sections";
+import { rule as noHttpPropsRule } from "./linter-rules/no-http-props";
+
+linter.register(
+  noHttpPropsRule,
+  noHeadinglessSectionsRule,
+  checkPunctuation,
+  localRefsExist,
+  checkInternalSlots
+);
+
+const coreDefaults = {
+  lint: {
+    "no-headingless-sections": true,
+    "no-http-props": true,
+    "check-punctuation": false,
+    "local-refs-exist": true,
+    "check-internal-slots": false,
+  },
+  pluralize: true,
+  specStatus: "base",
+  highlightVars: true,
+  license: "CC-BY",
+  addSectionLinks: true,
+};
+
+export default coreDefaults;
```

Comment:
I'm not sure we should export these... or we should only export them if there is a good reason to do so (e.g., some other module asks for them in the future for some valid reason).

Reply:
I think exporting anything without actual use should be avoided.

Reply:
agree,